### PR TITLE
Fixing Unicode error and other stuff

### PR DESCRIPTION
**Pull request by denilsonsa**
Fixing unicode error upon launch (also fixed in elecnix/screenkey@5c362f2aabd80011d9c518a978483ce3332b22a2, but using a different logic).

Other minor cleanup, mostly whitespace (add `?w=1` to GitHub URLs to ignore whitespace when viewing the diffs) and [code styling](https://www.python.org/dev/peps/pep-0008/) (with help of [pep8 tool](https://pypi.python.org/pypi/pep8)).

### CODE REVIEW

**Comment by seletskiy** [comment]
@denilsonsa: wow, thanks! Will merge soon.

File: Screenkey/listenkbd.py [hunk 1]
```diff
@@ -166,14 +167,14 @@ def update_text(self, string=None):
...
         # This is not the most efficient way to detect the
         # use of sudo/gksudo but it works.
         if not self.nosudo:
-            sudo_is_running = subprocess.call(['ps', '-C', 'sudo'],
-                        stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-            if not sudo_is_running:
+            sudo_is_running = 0 == subprocess.call(
+                ['ps', '-C', 'sudo'],
+                stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+            if sudo_is_running:
```
Comment (seletskiy) [inline Screenkey/listenkbd.py:177]: I guess there should be `if not ...`, because ps will return 0 code if process exist.

Comment (seletskiy) [inline Screenkey/listenkbd.py:177]: Got it... Never mind.

**Comment by denilsonsa** [comment]
Yeah, I inverted the `sudo_is_running` logic because it makes more sense now.

**Pull request merged**